src/assets/scripts/client/game/GameController.js

```diff
@@ -1,11 +1,14 @@
 import $ from 'jquery';
 import _forEach from 'lodash/forEach';
 import _has from 'lodash/has';
+import EventBus from '../lib/EventBus';
 import GameOptions from './GameOptions';
 import { round } from '../math/core';
+import { EVENT } from '../constants/eventNames';
 import { GAME_OPTION_NAMES } from '../constants/gameOptionConstants';
-import { SELECTORS } from '../constants/selectors';
 import { TIME } from '../constants/globalConstants';
+import { SELECTORS } from '../constants/selectors';
+import { THEME } from '../constants/themes';
 
 // Temporary const declaration here to attach to the window AND use as internal property
 const game = {};
```

Comment:
if there are no more `prop.game` values throughout the app, this line can be removed along with the first line in the `constructor`.